manim/cli/cfg/commands.py

```diff
@@ -225,8 +250,10 @@ def show() -> None:
         console.print("\n")
 
 
-def export(path):
-    if os.path.abspath(path) == os.path.abspath(os.getcwd()):
+@cfg.command(context_settings=CONTEXT_SETTINGS)
+@click.option("-d", "--dir", default=os.getcwd())
+def export(dir):
+    if os.path.abspath(dir) == os.path.abspath(os.getcwd()):
```

Comment:
Try using `pathlib` here

Reply:
maybe for a later PR.

Reply:
Yeah, I just copied these changes from `__main__.py`, I'll update it/remove the os import.